packages/docs/src/routes/docs/(qwik)/components/overview/index.mdx

```diff
@@ -113,40 +113,46 @@ export default component$(() => {
 
 ## Props
 
-Props are used to pass data from the parent into the component. Props are accessible via the `props` argument to the component$ function.
+Props are used to pass data from the parent into the component. They are accessible via the `props` argument to the `component$` function. Data passed as props is immutable, and cannot be changed. In cases where data sent as props needs to be updated, it is recommended to use signals.
 
-In this example a component `Item` declares optional `name`, `quantity`, `description`, and `price`.
+In the example below a component `Item` declares optional `name`, `quantity`, `description`, and `price` props. The `quantity` and `price` props are signals; `price` is a signal before being sent to `Item` and can be changed from the parent component. `quantity` is sent as a number value, and a new signal is created from the initial value inside of the `Item` component, where it can be changed.
```

Comment:
No, this is actually not a good idea. You should deference signals when you put them into props, and the optimizer will forward them.

Using signals as props is possible but it complicates the mental model and actual code without benefits.

Reply:
@codyroberts so in Qwik, there's an optimization happening behind the scenes which allows you to use `someProp={mySignal.value}` 

This won't actually create a subscription, but will pass the signal down for you and let you use that value wherever you need to use it and the subscription will be created for you.

@wmertens btw, it'll be "read only", meaning you can't really change that signal when you pass it's value as a prop.

If you want the ability to change it you need to pass it unwrapped (as the example shows) and then you can modify it.

Reply:
Oh I see now, thank you! 

If necessary I could probably make the documentation clearer about the fact that if you don't want to use a signal and don't need to modify the original value then you can create a local variable and set it to the props value, then change that.

Reply:
"I could probably make the documentation clearer" yes please

Reply:
> If necessary I could probably make the documentation clearer about the fact that if you don't want to use a signal and don't need to modify the original value then you can create a local variable and set it to the props value, then change that.

Thanks!
yeah most of the time if you just need the prop value you'd probably just de-structure it like:
```
component$(({something}: MyProps) => {
...
} )

// OR

const { something } = props;

```

That's what mostly we're seeing